app/components/channel_intro/index.js

```diff
@@ -43,14 +41,12 @@ function makeMapStateToProps() {
             }
 
             creator = getUser(state, currentChannel.creator_id);
-            postsInChannel = state.entities.posts.postsInChannel[currentChannel.Id];
         }
 
         return {
             creator,
             currentChannel,
             currentChannelMembers,
-            isLoadingPosts: (!postsInChannel || postsInChannel.length === 0) && getPostsRequestStatus === RequestStatus.STARTED,
```

Comment:
Why was receiving a push notification setting the status of the get posts request to started? Won't this make the channel intro message appear when the channel is still loading, before the user reaches the end of the post list?

Reply:
1. reason is cause now when we receive a push notification we do get the post of that channel to be able to show them as soon as possible so  `getPostsRequestStatus` was changed
2. It won't have an effect cause we have the footer showing a spinner, and then we mount the channel intro, you didn't noticed before as it was a spinner -> spinner -> intro, now it is spinner -> intro

Reply:
Sounds good. We might want to revisit the whole request store at some point since I don't think it's working out at all how we intended it since components never know if it's their request that triggered the state change, so you can easily get weird behaviour like that